common/workunit/workunit.cpp

```diff
@@ -5474,7 +5491,8 @@ void CWorkUnitFactory::importWorkUnit(const char *zapReportFileName, const char
         }
         void setUNZIPDir()
         {   //Set a unique unzip folder inside the component's data folder
-            unzipDir.append(importDir).append(PATHSEPSTR).append(wuid.get());
+            unzipDir.append(importDir);
+            addPathSepChar(unzipDir).append(wuid.get());
```

Comment:
NB: somewhat unrelated, but prior to this change, there would be a double // created in the path.